[2025-03-30T17:49:11.612+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-30T17:49:11.628+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_deliverable.create_hive_table manual__2025-03-30T17:47:45.099554+00:00 [queued]>
[2025-03-30T17:49:11.632+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_deliverable.create_hive_table manual__2025-03-30T17:47:45.099554+00:00 [queued]>
[2025-03-30T17:49:11.632+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2025-03-30T17:49:11.646+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): create_hive_table> on 2025-03-30 17:47:45.099554+00:00
[2025-03-30T17:49:11.651+0000] {standard_task_runner.py:64} INFO - Started process 242 to run task
[2025-03-30T17:49:11.653+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'dag_deliverable', 'create_hive_table', 'manual__2025-03-30T17:47:45.099554+00:00', '--job-id', '148', '--raw', '--subdir', 'DAGS_FOLDER/dag_deliverable.py', '--cfg-path', '/tmp/tmpk8vn_r5r']
[2025-03-30T17:49:11.654+0000] {standard_task_runner.py:91} INFO - Job 148: Subtask create_hive_table
[2025-03-30T17:49:11.680+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_deliverable.create_hive_table manual__2025-03-30T17:47:45.099554+00:00 [running]> on host airflow
[2025-03-30T17:49:11.732+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_deliverable' AIRFLOW_CTX_TASK_ID='create_hive_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-30T17:47:45.099554+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-30T17:47:45.099554+00:00'
[2025-03-30T17:49:11.732+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-30T17:49:11.749+0000] {base.py:84} INFO - Using connection ID 'hive_default' for task execution.
[2025-03-30T17:49:11.773+0000] {hive.py:475} INFO - USE `default`
[2025-03-30T17:49:11.795+0000] {hive.py:475} INFO - 
            CREATE EXTERNAL TABLE IF NOT EXISTS weather.sensor_data (
                record_timestamp DATE,
                temperature_salon FLOAT,
                humidity_salon FLOAT,
                air_salon FLOAT,
                temperature_chambre FLOAT,
                humidity_chambre FLOAT,
                air_chambre FLOAT,
                temperature_bureau FLOAT,
                humidity_bureau FLOAT,
                air_bureau FLOAT,
                temperature_exterieur FLOAT,
                humidity_exterieur FLOAT,
                air_exterieur FLOAT
            )
            ROW FORMAT SERDE 'org.apache.hive.hcatalog.data.JsonSerDe'
            STORED AS TEXTFILE
            LOCATION 'hdfs://namenode:9000/topics/sensores'
        
[2025-03-30T17:49:12.354+0000] {dag_deliverable.py:134} INFO - âœ… Tabla creada correctamente en Hive
[2025-03-30T17:49:12.355+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-03-30T17:49:12.356+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-30T17:49:12.360+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_deliverable, task_id=create_hive_table, run_id=manual__2025-03-30T17:47:45.099554+00:00, execution_date=20250330T174745, start_date=20250330T174911, end_date=20250330T174912
[2025-03-30T17:49:12.387+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-03-30T17:49:12.396+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-30T17:49:12.396+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
